redis/commands/graph/commands.py

```diff
@@ -214,6 +217,26 @@ def explain(self, query, params=None):
         plan = self.execute_command(EXPLAIN_CMD, self.name, query)
         return ExecutionPlan(plan)
 
+    def constraint(
+        self,
+        key: KeyT,
+        op: str,
+        type: str,
+        entity_type: str,
+        label_name: str,
+        n_props: int,
+        props: List[str]):
+        """
+        Constraint operation,
+        For more information see `GRAPH.CONSTRAINT <https://redis.io/commands/graph.constraint>`_. # noqa
```

Comment:
can I see the graph.constraint docs that are proposed? it's a dead link

Reply:
It won't be when we merge it

Reply:
@OfirMos did this happne, or can we close off the PR? I don't see it merged or available currently.